src/main/java/uk/gov/hmcts/reform/hmc/validator/LinkedHearingValidator.java

```diff
@@ -366,8 +366,8 @@ public LocalDate filterHearingResponses(HearingEntity hearingEntity) {
         if (log.isDebugEnabled()) {
             if (hearingResponse.isPresent()) {
                 log.debug("hearing response: {} : {}",
-                        hearingResponse.get().getHearingResponseId(),
-                        hearingResponse.get().getRequestTimeStamp());
+                          hearingResponse.get().getHearingResponseId(),
```

Comment:
No changes in the code. please remove the extra spaces.

Reply:
so strange behaviour when upgrading cftlib, these are required or style checks fail